js_modules/dagster-ui/packages/ui-core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -151,6 +240,13 @@ export const AssetsCatalogTable = ({
     <AssetTable
       view={view}
       assets={displayed}
+      isFiltered={
+        !!(
+          filters.changedInBranch?.length ||
+          filters.computeKindTags?.length ||
+          filters.groups?.length
```

Comment:
I'm not a big fan of this because it's easy to forget to add new items to this check. I wonder if we could do this in another way? Maybe `Object.keys(filters).length` or `!isEqual(filters, defaultFilters)`?